### PR TITLE
render caption in table if passed in as a prop

### PR DESCRIPTION
**Pull request by jhp0621**
Allow table to have a caption specified: https://github.com/LaunchPadLab/lp-components/issues/463

## Author Checklist

- [ ] Add unit test(s)
- [ ] Update version in package.json (see the [versioning guidelines](https://github.com/LaunchPadLab/opex-public/blob/main/gists/npm-package-guidelines.md#pull-requests-and-deployments))
- [ ] Update documentation (if necessary)
- [ ] Add story to storybook (if necessary)
- [ ] Assign dev reviewer

### CODE REVIEW

**Comment by jhp0621** [comment]
opening a new PR since it was gonna be hard to undo formatting in https://github.com/LaunchPadLab/lp-components/pull/513

File: src/tables/sortable-table.js [hunk 1]
```diff
@@ -148,41 +151,49 @@ function SortableTable({
   }
 
   return (
-    <table className={classnames(className, { 'sortable-table': !disableSort })} {...filterInvalidDOMProps(rest)}>
-      <thead>
-        <tr>
-          {columns.map((column, key) => {
-              const Header = column.headerComponent || headerComponent || DefaultHeader
+    <>
+      {caption && <caption>{caption}</caption>}
```
Comment (jhp0621) [inline src/tables/sortable-table.js:155]: this is the only change in return

File: migration-guides/v6.0.0.md [hunk 2]
```diff
@@ -269,4 +271,25 @@ Replace `hideCloseButton` with `preventClose`. If you still need the modal to cl
...
+
+## 13. `<Table />` and `<SortableTable />` components now render a `<caption>` element with a `caption` prop it receives
+
+`caption` is a node prop and defaults to null. <caption> html tag is only rendered if `caption` is present.
+```jsx
+// Before
+<caption>My Awesome Table</caption>
+<Table>
+  Table content
+</Table>
```
Comment (chawes13) [inline migration-guides/v6.0.0.md:284]: I don't think this is an appropriate Before example. The caption tag must be rendered **within** the table tag
Image 2022-06-14 at 9 46 37 AM

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption

File: src/tables/sortable-table.js (hunk 1 above)
Comment (chawes13) [reply src/tables/sortable-table.js:155]: This needs to be rendered as the first child of the `<table>`. See previous comment for source

**Review by chawes13 — CHANGES_REQUESTED** [review]
Can you also add tests?

**Comment by chawes13** [comment]
@jhp0621 I decided against including information in the migration guide because this is not a breaking change. I also updated the placement and added a few tests. Let me know if you have any questions!

**Review by chawes13 — APPROVED** [review]

**Pull request merged**